experimental/HttpBenchmark/runtimeconfig.template.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Replaced by the MSBuild setting `<ServerGarbageCollection>true</ServerGarbageCollection>`